src/wampy.js

```diff
@@ -394,13 +393,13 @@ class Wampy {
      * @private
      */
     _log (...args) {
-        if (this._options.debug) {
-            if (this._options.logger) {
-                this._options.logger(args);
-            } else {
-                console.log('[wampy]', args);
-            }
+        if (!this._options.debug) { return; }
+
+        if (this._options.logger) {
+            return this._options.logger(args);
```

Comment:
Any reason to return another call instead of just omitting `return` statement and thus returning undefined?
Below the same one.

Reply:
Hi, it's called the [Return Early Pattern](https://medium.com/swlh/return-early-pattern-3d18a41bba8).  they're there as a way to exit early and not have to evaluate any logic after the call has been returned. By doing it like this we're also able to remove the `else` block at the bottom, making the method smaller.

I know it doesn't make that much of a difference in this specific function because it's already small, but I did it here first to see it it's a welcome change, so that I can apply this pattern to some bigger functions in future PRs, where the benefits would be a lot more noticeable. 

Let me know what you think!

Reply:
Ah okay! I know that one, just want to ask maybe you mean/know something different :) 
As `_log` is an internal one and there is nothing done with its return results, I think we can safely implement it. 
So you are welcome to introduce such improvements!